src/v2/Apps/Auctions/AuctionsApp.tsx

```diff
@@ -25,36 +31,44 @@ const AuctionsApp: React.FC<AuctionsAppProps> = props => {
   return (
     <AppContainer>
       <AuctionsMeta />
-      <HorizontalPadding>
-        <Text mt={3} mb={1} variant="largeTitle">
-          Auctions
-        </Text>
-        <Text py={1}>
-          Bid on works you love with auctions on Artsy. With bidding opening
-          daily, Artsy connects collectors like you to art from leading auction
-          houses, nonprofit organizations, and sellers across the globe. We
-          feature premium artworks including modern, contemporary, and street
-          art, so you can find works by your favorite artists—and discover new
-          ones—all in one place.
-        </Text>
-        <RouterLink
-          to="https://support.artsy.net/hc/en-us/sections/360008298773-Bid-at-Auction"
-          noUnderline
-        >
-          <Text variant="mediumText" py={1}>
-            Learn more about bidding on Artsy{" "}
-            <ChevronIcon
-              title={null}
-              direction="right"
-              color="black"
-              height="15px"
-              width="14px"
-              top="3px"
-              left="3px"
-            />
-          </Text>
-        </RouterLink>
-      </HorizontalPadding>
+      <GridColumns>
+        <Column span={6}>
```

Comment:
I'd double check mobile layout in chrome dev tools and make sure that we're collapsing this into a 12 column. So something llike `span={[12, 6]}` (and same below)

Reply:
Updated.